### DISCUSSION TITLE
Hint icon location when using inline label

### DISCUSSION THREAD

**Discussion by rhysflook** [filamentphp/filament#9498] [Help] [answered]
When using the `inlineLabel` and the `hintIcon` method on a form field, the icon appears to the left of the input element rather than next the the hint itself.

I can make the icon render next to the hint by putting the svg code for the icon in the hint text and rendering that as html, but I was wondering if there is built-in a way to define the location?

**Comment by leandrocfe** [comment]
I think it isn't possible, but where would you like to show the icon? Could you show an image/screenshot please?

**Reply by rhysflook** [reply]
![image](https://github.com/filamentphp/filament/assets/53523326/39605476-bfe3-4247-8a28-85ea321b2699)

So this is what I am going for, with the icon to the left of the helperText. Forgot to mention I am using `helperText` in my original question.

**Reply by leandrocfe** [reply, marked as answer]
What do you think?
![image](https://github.com/filamentphp/filament/assets/3833889/9a8f7d3b-afc4-4dee-a5df-6a5676c8c3aa)


```php
Forms\Components\TextInput::make('field')
    ->helperText(view('custom-helper-text', [
        'icon' => 'heroicon-o-information-circle',
        'text' => 'This is a custom helper text'
    ]))
```

```html
<!-- resources/views/custom-helper-text.blade.php -->
<div class="flex gap-2">
    <div class="h-5 w-5">{{ svg($icon) }}</div>
    <div>{{ $text }}</div>
</div>
```

**Reply by rhysflook** [reply]
Honestly didn't even realise you can use the view helper in this way, thanks!

**Reply by leandrocfe** [reply]
It could be better, but it works!

**Comment by sinnbeck** [comment]
Shouldn't that be `->hint()` instead of `->inlineLabel()` if they should be together and be a hint ?

**Reply by rhysflook** [reply]
Forgot the important detail that I am using `helperText` in my original question, my bad. 

This is the layout I need to make, it is doable but just a bit messy so I was wondering if there is anything built in to filament to handle this case.

![image](https://github.com/filamentphp/filament/assets/53523326/eb875df3-5877-49d2-8c49-bc8a52bff509)
